lib/car.rb

```diff
@@ -0,0 +1,21 @@
+class Car
+  attr_reader :make, :model, :monthly_payment, :loan_length, :color
+
+  def initialize(type, monthly_payment, loan_length)
+    @type = type
+    @make = @type.split(" ").first
+    @model = @type.split(" ").last
+    @monthly_payment = monthly_payment
+    @loan_length = loan_length
+    @color = nil
+  end
```

Comment:
Good work using `split` on your `make_model` argument. Can you think of a different way to implement this to prevent having to call `split` multiple times within `initialize`?